.github/workflows/ToggleLockBranch.yml

```
@@ -15,6 +15,9 @@ jobs:
         with:
           secrets: |
             development/github/token/{REPO_OWNER_NAME_DASH}-lock token | lock_token;
+            development/kv/data/slack token | slack_api_token;
       - uses: sonarsource/gh-action-lt-backlog/ToggleLockBranch@v1
         with:
           github-token: ${{ fromJSON(steps.secrets.outputs.vault).lock_token }}
+          slack-token: ${{ fromJSON(steps.secrets.outputs.vault).slack_api_token }}
+          slack-channel: team-lang-dotnet-autoscan
```

To remove my confusion
```suggestion
          slack-channel: team-lang-dotnet-s4net
```